packages/astro/src/core/build/page-data.ts

```diff
@@ -71,30 +71,18 @@ export async function collectPagesData(
 				css: new Set(),
 				hoistedScript: undefined,
 				scripts: new Set(),
-				preload: await ssrPreload({
```

Comment:
This removes the "preload" step which was the expensive part of the build that we no longer need to do. Metadata is extracted via the compiler instead.